File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/businessmanagement/cs/convertopportunitytoquote1.cs
// <snippetconvertopportunitytoquote1>


 // Convert an opportunity to quote.
 GenerateQuoteFromOpportunityRequest quoteRequest = new GenerateQuoteFromOpportunityRequest()
 {
     // Columns that will be transferred
     ColumnSet = new ColumnSet("name", "customerid"),
     OpportunityId = _opportunityId
 };

 GenerateQuoteFromOpportunityResponse quoteResponse =
     (GenerateQuoteFromOpportunityResponse)_serviceProxy.Execute(quoteRequest);

 _quoteId = quoteResponse.Entity.Id;

// </snippetconvertopportunitytoquote1>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/businessmanagementvb/vb/convertopportunitytoquote1.vb
' <snippetconvertopportunitytoquote1>


 ' Convert an opportunity to quote.
 Dim quoteRequest As New GenerateQuoteFromOpportunityRequest() With { _
     .ColumnSet = New ColumnSet("name", "customerid"), .OpportunityId = _opportunityId}
     ' Columns that will be transferred

 Dim quoteResponse As GenerateQuoteFromOpportunityResponse = _
     CType(_serviceProxy.Execute(quoteRequest), GenerateQuoteFromOpportunityResponse)

 _quoteId = quoteResponse.Entity.Id

' </snippetconvertopportunitytoquote1>